README.md

```diff
@@ -454,6 +454,37 @@ For example:
 
 See [JsonPathFilterQuery JavaDoc](oauth-common/src/main/java/io/strimzi/kafka/oauth/jsonpath/JsonPathFilterQuery.java) for more information about the syntax.
 
+###### Group extraction
+
+When using custom authorization (by installing a custom authorizer) you may want to take user's group membership into account when making the authorization decisions.
+One way is to obtain and inspect a parsed JWT token from `io.strimzi.kafka.oauth.server.OAuthKafkaPrincipal` object available through `AuthorizableRequestContext` passed to your `authorize()` method.
+Another way is to configure group extraction at authentication time, and get groups as a set of principals from `OAuthKafkaPrincipal` object.
+There are two configuration parameters for configuring group extraction:
```

Comment:
Should we provide some guidance about when one is preferable to the other?

Reply:
This just depends on what the custom authorizer requirements are. It may be that the developer just needs a list of groups to make the authorization decision. Or it may need some other claims that are only available in the JWT token. I don't really have an idea what more to add.

Reply:
I did rework the text a little bit.